python/ray/tests/test_actor_retry_2.py

```diff
@@ -338,5 +339,39 @@ def test_task_retries_on_exit(ray_start_regular_shared):
     ]
 
 
+def test_retry_not_dependent_task_seq_no(ray_start_regular_shared):
+    """
+    1. one is submitted
+    2. two is submitted
+    3. the first attempt of one fails
+    4. the second attempt of one is submitted
+    5. two succesfully finishes
```

Comment:
the comment is not really intelligible right now without first reading the code and then trying to reverse engineer it. make it more precise.